e2e/screenshots/screenshots.steps.ts

```diff
@@ -0,0 +1,105 @@
+import { binding, given, then, when, before, after } from 'cucumber-tsflow';
+import { CallbackStepDefinition } from 'cucumber';
+import { ConnectionsListComponent } from '../connections/list/list.po';
+import { World, expect, P } from '../common/world';
+import { browser } from 'protractor';
+import { log } from '../../src/app/logging';
+import * as fs from 'fs';
+
+/**
+ * Created by mmelko.
+ */
+@binding([World])
+class ScreenshotsSteps {
+
+    location: string;
+    resolutions: Array<{
+        'height': number,
+        'width': number,
+    }>;
+
+    constructor(private world: World) {
+        this.location = 'e2e/cucumber-reports/screenshots/';
+        this.resolutions = new Array();
+    }
+
+    @before()
+    public init(): void {
+        if (!fs.existsSync(this.location)) {
+            fs.mkdirSync(this.location);
+        }
+    }
```

Comment:
Maybe I would prefer something like clean up of existing screenshots or move them e.g. to archive dir. To prevent pilling up of img files between local reruns.